usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -85,6 +95,15 @@ def handle(self, *args, **options):
         self.config['recreate'] = options['recreate']
         self.config['stale'] = options['stale']
         self.config['keep'] = options['keep']
+        self.config['index_name'] = options['index_name']
+        self.config['previous_index'] = options['previous_index']
+
+        mappingfile = os.path.join(settings.BASE_DIR, 'usaspending_api/etl/es_transaction_mapping.json')
+        with open(mappingfile) as f:
+            mapping_dict = json.load(f)
+            self.config['mapping'] = json.dumps(mapping_dict)
+        self.config['doc_type'] = str(list(mapping_dict['mappings'].keys())[0])
```

Comment:
Is there a better way to do this? It seems fragile, I could see someone modifying the json file without understanding the ramifications.

If not please add a comment of intended behavior.

Reply:
Then they shouldn't modify the json file. Single type mappings for our version of ES .. but will modify code to check. https://www.elastic.co/guide/en/elasticsearch/reference/master/removal-of-types.html